configure.ac

```diff
@@ -38,8 +38,8 @@ AM_INIT_AUTOMAKE(
   1.11.1
   dnl Automake version before 1.13 (when the serial-tests option was
   dnl still the default) still defined the badly obsolete macro
-  dnl 'AM_PROG_INSTALL'.
-  m4_ifndef([AM_PROG_INSTALL], [serial-tests])
+  dnl 'AC_PROG_INSTALL'.
+  m4_ifndef([AC_PROG_INSTALL], [serial-tests])
```

Comment:
This change is not correct, and appears to have been done by some sort of automated process that does not understand the code it touched.

Reply:
I apologise. I encountered errors when trying to build OpenARC, and changes from [this commit](https://github.com/trusteddomainproject/OpenDKIM/pull/163/files) to OpenDKIM did the trick: I could now build OpenARC.

Since that pull request had been committed to similar software by the TDP (the two configure.ac files look very similar), I presumed that the changes were both correct, and done properly. This is also why I was so bold as to open a pull request, instead of an Issue. Again, I apologise for using your time and will use this opportunity to learn.

I will be happy to revert this change, and adapt the below.

Reply:
What errors did you encounter? I tried building with autoconf 2.71 and had no issues. There were the expected warnings about obsolete macros, but no failures.

This change was also incorrect in OpenDKIM but I'm not surprised no one noticed, since it's kind of subtle and might only break very old environments. The intent of this check is to only pass `[serial-tests]` to versions of Automake where that's a supported parameter, by checking whether an obsolete macro is defined. The automated tool did a search and replace for the obsolete macro, replacing it with a non-obsolete one and making the check incorrect.

See https://github.com/flowerysong/OpenARC/commit/0676227216f69bb52a4da2e52faf1978062b1fd7 for how I would approach these updates.

Reply:
Thank you, @flowerysong. I appreciate your time and resourcefulness! I like the look of your proposed changes. I see you are keeping `LT_INIT`; and that you updated the AC_OUTPUT (and config files), besides dealing with the obsolete macros.

Where do you want to go from here (can I copy your changes, or do you want to do a PR yourself)?

I built OpenARC again. There are no errors during the autoconf stage or build stage, only warnings. My mistake. I went down the rabbit hole after seeing the following two warnings during 'make install':
 - libtool: warning: remember to run 'libtool --finish /usr/lib'
 - libtool: warning: '../libopenarc/libopenarc.la' has not been installed in '/usr/lib'

If relevant, I can post here (or in an issue rather?) the complete logs?